cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+from time import sleep
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.wait_for_db()
+
+    def wait_for_db(self):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.ensure_connection()
+                self.stdout.write(self.style.SUCCESS("Database available!"))
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                sleep(1)
```

Comment:
The while loop here is used to continuously check if the database connection is available. It's a good approach to handle situations where the database service might not be ready when the Django service starts, especially when using Docker to manage your services. However, please be aware that this could potentially lead to an infinite loop if the database service is not set up correctly or if there is a network issue. You might want to consider adding a timeout or maximum number of retries to avoid this scenario.